nano/node/socket.cpp

```diff
@@ -276,22 +271,45 @@ void nano::server_socket::on_connection (std::function<bool (std::shared_ptr<nan
 			}
 			else
 			{
-				this_l->node.logger.try_log ("Network: Unable to accept connection: ", ec_a.message ());
+				this_l->node.logger.always_log ("Network: Unable to accept connection: ", ec_a.message ());
 			}
 
-			// If the callback returns true, keep accepting new connections
-			if (callback_a (new_connection, ec_a))
+			if (this_l->is_temporary_error (ec_a))
 			{
 				this_l->on_connection (callback_a);
```

Comment:
Needs to call it in same way as other case:
this_l->node.stats.inc (nano::stat::type::tcp, nano::stat::detail::tcp_accept_failure, nano::stat::dir::in);
						boost::asio::post (this_l->strand, boost::asio::bind_executor (this_l->strand, [this_l, callback_a] () {
							this_l->on_connection (callback_a);
						}));